keyboards/ergotaco/keymaps/default/keymap.c

```diff
@@ -0,0 +1,43 @@
+/* Good on you for modifying your layout! if you don't have 
+ * time to read the QMK docs, a list of keycodes can be found at
+ *
+ * https://github.com/qmk/qmk_firmware/blob/master/docs/keycodes.md
+ *
+ * There's also a template for adding new layers at the bottom of this file!
+ */
+
+#include QMK_KEYBOARD_H
+
+#define IGNORE_MOD_TAP_INTERRUPT
```

Comment:
Already defined at the keyboard level in `config.h`, and to my knowledge, defining it here wouldn't work under any circumstance.

```suggestion
```